app/policies/trainee_policy.rb

```diff
@@ -10,13 +10,16 @@ def initialize(user, scope)
     end
 
     def resolve
-      user.system_admin? ? scope.all : scope.where(provider_id: user.provider_id)
+      user.system_admin? ? scope.all : scope.where(provider_id: user.provider_id).kept
```

Comment:
```suggestion
      user.system_admin? ? scope.kept : scope.where(provider_id: user.provider_id).kept
```
I think this should extend to even system admins (i.e: we shouldn't be showing the discarded trainees in the UI).
This will also affect the statistics if we don't

Reply:
agreed to show all records to system admins for now.